must-gather/collection-scripts/gather_sandboxed_containers_logs

```diff
@@ -4,11 +4,24 @@ IFS=$'\n'
 
 BASE_COLLECTION_PATH="/must-gather"
 
-# kataconfig
 OSC_PATH=${BASE_COLLECTION_PATH}/sandboxed-containers
 mkdir -p ${OSC_PATH}
 
-oc describe kataconfig > "${OSC_PATH}/kataconfig_description"
+operator_resources=()
+operator_resources+=(kataconfig)
+operator_resources+=(clusterserviceversion)
+operator_resources+=(subscription)
+operator_resources+=(services)
+
+for operator_resource in "${operator_resources[@]}"; do
+	oc describe ${operator_resource} -n openshift-sandboxed-containers-operator > "${OSC_PATH}/${operator_resource}_description"
+done
+
+# NOTE Jul 16, 2021: a single one is expected as of this writing, with more
+# possibly being added in the future
+KATA_WEBHOOK_CONFIGS=$(/usr/bin/oc get validatingwebhookconfigurations -A -o'custom-columns=name:metadata.name' --no-headers | grep kata)
+/usr/bin/oc describe validatingwebhookconfigurations ${KATA_WEBHOOK_CONFIGS} > "${OSC_PATH}/validatingwebhookconfigurations_description"
```

Comment:
I think we're super fine just using `oc` rather than `/usr/bin/oc`, and the usage of `oc` (merely) is consistent throughout the whole file.

Reply:
@fidencio Just pushed a fixed version.  Good catch, the absolute paths were just an oversight, they were supposed to be all removed at this point - my bad!